src/simplification/SimplifyItems.ts

```diff
@@ -12,50 +13,66 @@ type Output = CommonModel | undefined;
  * @param seenSchemas already seen schemas and their corresponding output, this is to avoid circular schemas
  */
 export default function simplifyItems(schema: Schema | boolean, simplifier : Simplifier, seenSchemas: Map<any, Output> = new Map()) : Output {
-  let tempOutput: Output;
   if (typeof schema !== 'boolean') {
-    if (seenSchemas.has(schema)) return seenSchemas.get(schema)!;
+    if (seenSchemas.has(schema)) return seenSchemas.get(schema);
+    const tempOutput: Output = new CommonModel();
```

Comment:
Why `tempOutput` name?

Reply:
🤷 you don't like it?  😄

Reply:
`temp` prefix is always used in small function and not for returned values, here you return this var. If you have no idea for a different name, leave it :)

Reply:
True, I will change it.